src/middleware/validators.js

```diff
@@ -6,67 +6,85 @@ const wlogger = require('../lib/wlogger')
 const KeyEncoder = require('key-encoder').default
 const keyEncoder = new KeyEncoder('secp256k1')
 
+const jwtOptions = {
+  algorithms: ['ES256']
+}
+
```

Comment:
This the essential stuff that was missing with the validation. This looks correct according to the [jsonwebtoken documentation](https://www.npmjs.com/package/jsonwebtoken#usage), but it's failing with this error:

`Err: Token could not be decoded: Error: PEM_read_bio_PUBKEY failed`